src/element/collision.ts

```diff
@@ -656,12 +669,7 @@ const pointInBezierEquation = (
   return false;
 };
 
-const hitTestCurveInside = (
-  drawable: Drawable,
-  x: number,
-  y: number,
-  lineThreshold: number,
```

Comment:
Unrelated: How is the threshold being applied? (cause it seems that it is from playing with the UI).

Reply:
It doesn't apply a threshold here. This function is detecting area within the shape. The threshold you're seeing is coming from fallback to detecting shapes outline curves.